### ISSUE TITLE
Output in CSV format

### ISSUE THREAD

**Issue by amoghsg** [redsolver/dart_license_checker#9]
It will be really helpful if the output is in CSV or json format that can be consumed programatically.

**Comment by Dieterbe** [comment]
yes, especially because sometimes one package uses multiple licenses. for example https://pub.dev/packages/vector_math/license uses zlib + BSD license. this is hard to parse out of the output with shellscript
